src/main/java/eu/europeana/metis/sandbox/config/ElasticConfig.java

```diff
@@ -0,0 +1,46 @@
+package eu.europeana.metis.sandbox.config;
+
+import static java.util.stream.Collectors.joining;
+
+import co.elastic.apm.attach.ElasticApmAttacher;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "elastic")
```

Comment:
Adding this here:
I think perhaps the "sandbox.metrics.dataset.frequency" should be part of the "elastic" structure?
It is related specifically to that after all.
And also probably the "dataset" should be part of the name since we have metrics in general.
Somthing like "elastic.metrics.frequency" ?